assets/less/variables.less

```diff
@@ -17,6 +17,12 @@
 
 @detailGray: #f7f7f7;
 
+// Night mode
+@nightGray: #1A1D26;
+@nightCodeBlue: #4B556E;
+@nightMediumBlue: #69778C;
+@nightPurple: #CEB4ED;
+
```

Comment:
Is this comment a mistake? I'm using those variables. Or are you suggesting I move them to a variables.less file inside the night folder?